docs/user-guide/01-blueprint-reference.md

```diff
@@ -1243,6 +1245,121 @@ The following mountpoints are allowed, but their sub-directories are **not** all
 
 - `/usr`
 
+### Disk 🔵 🟤 🟣 {#disk}
```

Comment:
Should we mention which version of composer do you need? Maybe even match the RHEL versions (RHEL 9.6/RHEL 10). And I suggest doing the same on the Partitioning page. People will get confused otherwise I think (because the new partitioning is so cool).

btw is this actually available in osbuild-composer? Don't we need to do blueprint translation there? 🤔